pkg/metrics/collector.go

```diff
@@ -47,6 +48,35 @@ var (
 			return vs.UnknownCount
 		},
 	}
+	exposedSecretLabels = []string{
+		"namespace",
```

Comment:
probably not must for this PR , maybe we should make const from all the repeating fields name(namespace,name,severity and etc)

Reply:
yup, will create separate PR for it.